src/cmd/services/m3dbnode/config/bootstrap.go

```diff
@@ -195,6 +204,12 @@ func ValidateBootstrappersOrder(names []string) error {
 			bfs.FileSystemBootstrapperName,
 			peers.PeersBootstrapperName,
 		},
+		uninitialized.UninitializedBootstrapperName: []string{
```

Comment:
Looks good.  For peers bootstrapper, shouldn't the commit log bootstrapper name also appear in there (so it can appear after the commit log bootstrapper as well)?

Reply:
good call, will fix